git-setup.sh

```diff
@@ -35,3 +39,4 @@ for module in "$@"; do
     fi
   fi
 done
+git restore --staged .gitmodules
```

Comment:
Adding a git submodule also adds this file which is in .gitignore, unstage this as well.